src/main/resources/dbmigration/dropTable.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE employee;
```

Comment:
please use TRUNCATE TABLE instead of DROP TABLE

Reply:
truncate doesn't work because a new table is created before each test (I don't know how to make it appear once before all tests)